### PR TITLE
set public API docs page title correctly

### PR DESCRIPTION
**Pull request by MrGrinst**

### CODE REVIEW

File: src/Aquifer.Public.API/Program.cs [hunk 1]
```diff
@@ -48,7 +48,11 @@
         config.Endpoints.Configurator = ep => ep.AllowAnonymous();
     })
     .UseOpenApi()
-    .UseReDoc(options => options.Path = "/docs");
+    .UseReDoc(options =>
+    {
+        options.Path = "/docs";
+        options.DocumentTitle = "Aquifer API Docs";
```
Comment (jwinston-bn) [inline src/Aquifer.Public.API/Program.cs:54]: Is this removing the (1.0.0) piece? Would prefer "Documentation" over "Docs".

Comment (MrGrinst) [reply src/Aquifer.Public.API/Program.cs:54]: It's setting the page title so it shows right in the browser tab bar:
image

Comment (MrGrinst) [reply src/Aquifer.Public.API/Program.cs:54]: `Documentation` feels a bit long for the page title, we could also just drop it entirely and do `Aquifer API`. Thoughts?

Comment (jwinston-bn) [reply src/Aquifer.Public.API/Program.cs:54]: I think it's pretty common for page titles to be that length or longer (I don't have a single tab open right now that isn't as long). If there's a quick and similar way to get rid of that "(1.0.0)" as part of this PR that'd be great.

Comment (MrGrinst) [reply src/Aquifer.Public.API/Program.cs]: Addressed this

**Review by jwinston-bn — APPROVED** [review]

**Pull request merged**